### PR TITLE
Added Visual Studio 2022 Community Preview 2.1

### PR DESCRIPTION
**Pull request by bearmannl**
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] Have you validated your manifest locally with `winget validate --manifest <path>`? 
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.0 schema](https://github.com/microsoft/winget-cli/blob/master/doc/ManifestSpecv1.0.md)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/22305)

### CODE REVIEW

File: manifests/m/Microsoft/VisualStudio/2022/Community-Preview/17.0.0 Preview 2/Microsoft.VisualStudio.2022.Community-Preview.locale.en-US.yaml [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+PrivacyUrl: https://privacy.microsoft.com/
+PackageName: Visual Studio Community 2022 Preview
+PackageUrl: https://visualstudio.microsoft.com/vs/preview/vs2022/
+License: Microsoft Pre-Release Software License
+LicenseUrl: https://visualstudio.microsoft.com/license-terms/vs2022-prerelease/
+Copyright: © 2021 Microsoft
+CopyrightUrl: https://www.microsoft.com/en-us/legal/copyright
+ShortDescription: Visual Studio 2022 Community Preview
+Description: Visual Studio Community is a free, fully-featured IDE for students, open-source contributors, and individual developers.
+Moniker: vs2022-Community-preview
```
Comment (ItzLevvie) [inline manifests/m/Microsoft/VisualStudio/2022/Community-Preview/17.0.0 Preview 2/Microsoft.VisualStudio.2022.Community-Preview.locale.en-US.yaml:19]:
```suggestion
Moniker: vs2022-community-preview
```

File: manifests/m/Microsoft/VisualStudio/2022/Community-Preview/17.0.0 Preview 2/Microsoft.VisualStudio.2022.Community-Preview.locale.en-US.yaml [hunk 2]
```diff
@@ -0,0 +1,28 @@
+# Created using wingetcreate 0.3.0.3
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.0.0.schema.json
+
+PackageIdentifier: Microsoft.VisualStudio.2022.Community-Preview
+PackageVersion: 17.0.0 Preview 2
```
Comment (ItzLevvie) [inline manifests/m/Microsoft/VisualStudio/2022/Community-Preview/17.0.0 Preview 2/Microsoft.VisualStudio.2022.Community-Preview.locale.en-US.yaml:5]:
```suggestion
PackageVersion: 17.0.0 Preview 2.1
```

File: manifests/m/Microsoft/VisualStudio/2022/Community-Preview/17.0.0 Preview 2/Microsoft.VisualStudio.2022.Community-Preview.installer.yaml [hunk 3]
```diff
@@ -0,0 +1,56 @@
+# Created using wingetcreate 0.3.0.3
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.0.0.schema.json
+
+PackageIdentifier: Microsoft.VisualStudio.2022.Community-Preview
+PackageVersion: 17.0.0 Preview 2
```
Comment (ItzLevvie) [inline manifests/m/Microsoft/VisualStudio/2022/Community-Preview/17.0.0 Preview 2/Microsoft.VisualStudio.2022.Community-Preview.installer.yaml:5]:
```suggestion
PackageVersion: 17.0.0 Preview 2.1
```

File: manifests/m/Microsoft/VisualStudio/2022/Community-Preview/17.0.0 Preview 2/Microsoft.VisualStudio.2022.Community-Preview.locale.en-US.yaml [hunk 4]
```diff
@@ -0,0 +1,28 @@
...
+PublisherUrl: https://www.microsoft.com/
+PublisherSupportUrl: https://visualstudio.microsoft.com/vs/support/
+PrivacyUrl: https://privacy.microsoft.com/
+PackageName: Visual Studio Community 2022 Preview
+PackageUrl: https://visualstudio.microsoft.com/vs/preview/vs2022/
+License: Microsoft Pre-Release Software License
+LicenseUrl: https://visualstudio.microsoft.com/license-terms/vs2022-prerelease/
+Copyright: © 2021 Microsoft
+CopyrightUrl: https://www.microsoft.com/en-us/legal/copyright
+ShortDescription: Visual Studio 2022 Community Preview
```
Comment (ItzLevvie) [inline manifests/m/Microsoft/VisualStudio/2022/Community-Preview/17.0.0 Preview 2/Microsoft.VisualStudio.2022.Community-Preview.locale.en-US.yaml:17]:
```suggestion
ShortDescription: Visual Studio Community 2022 Preview
```

File: manifests/m/Microsoft/VisualStudio/2022/Community-Preview/17.0.0 Preview 2/Microsoft.VisualStudio.2022.Community-Preview.yaml [hunk 5]
```diff
@@ -0,0 +1,9 @@
+# Created using wingetcreate 0.3.0.3
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.version.1.0.0.schema.json
+
+PackageIdentifier: Microsoft.VisualStudio.2022.Community-Preview
+PackageVersion: 17.0.0 Preview 2
```
Comment (ItzLevvie) [inline manifests/m/Microsoft/VisualStudio/2022/Community-Preview/17.0.0 Preview 2/Microsoft.VisualStudio.2022.Community-Preview.yaml:5]:
```suggestion
PackageVersion: 17.0.0 Preview 2.1
```

**Review by ItzLevvie — CHANGES_REQUESTED** [review]

File: manifests/m/Microsoft/VisualStudio/2022/Community-Preview/17.0.0 Preview 2/Microsoft.VisualStudio.2022.Community-Preview.installer.yaml (hunk 3 above)
Comment (bearmannl) [reply manifests/m/Microsoft/VisualStudio/2022/Community-Preview/17.0.0 Preview 2/Microsoft.VisualStudio.2022.Community-Preview.installer.yaml]: Good catch, thanks!

**Review by ItzLevvie — APPROVED** [review]

**Review by ItzLevvie — APPROVED** [review]

**Pull request merged**